pkg/systemd/logs.js

```diff
@@ -288,9 +288,11 @@ $(function() {
             }
         }
 
-        if (prio_level) {
+        if (!isNaN(prio_level)) {
             for (var i = 0; i <= prio_level; i++)
                 match.push('PRIORITY=' + i.toString());
+        } else {
+            console.warn('Priority level was incorrectly set to ', prio_level);
```

Comment:
I 'll remove this warning. In case priority was Everything prio_level is not a number but still valid.